ios/RCTTWVideoModule.m

```diff
@@ -143,8 +149,9 @@ - (void)addParticipantView:(TVIVideoView *)view sid:(NSString *)sid trackSid:(NS
 }
 
 RCT_EXPORT_METHOD(startLocalVideo) {
+  self.cameraOrientationTracker = [[CameraVideoOrientationTracker alloc] initWithOrientation:AVCaptureVideoOrientationPortrait];
```

Comment:
@andreychernikovich is this the correct default orientation also for iPad app? the override of `self.cameraOrientationTracker` is done for both platforms so need to make sure this doesn't break something in iPad. I can test if you want (but I have env issues).